pytorch_lightning/trainer/trainer.py

```diff
@@ -168,6 +172,18 @@ def __init__(
 
                     trainer = Trainer(early_stop_callback=early_stop_callback)
 
+            callback (:class:`.Callback`): Add a list of callbacks.
```

Comment:
The :class annotation can be omitted because sphinx will autogenerate one given the typehint above

Reply:
I also just saw right now that the documentation refers to "callback" but the actual argument is plural, callbacks.